docs/src/reference.md

```diff
@@ -26,8 +26,8 @@ MLFlowArtifactDirInfo
 createexperiment
 getexperiment
 getorcreateexperiment
-listexperiments
```

Comment:
instead of removing this, can we have it deprecarted for a version or two first? and then remove it?